src/App.js

```diff
@@ -1,18 +1,53 @@
 import React, { Component } from "react";
+import logo from "./logo.svg";
 import "./App.css";
+import evolutions from "evolutions";
+import pokedex from "./pokedex";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchInput: ""
+    };
+  }
+  handleEvolve = e => {
+    e.preventDefault();
+    console.log("evolve button clicked");
+    let evo = this.state.searchInput;
```

Comment:
Can this be re-written to 
```javascript 
    let evo = this.state.searchInput.toLowerCase();
```
All the keys in pokedex are in lowercase. It will throw an error when for example the input is "Pikachu" instead of "pikachu"

Reply:
Yes!